docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md

```diff
@@ -0,0 +1,109 @@
+| [ADS115 ADC] | Amplifier | [36.18 RON](https://www.sigmanortec.ro/modul-convertor-analogic-la-digital-adc-ads1115-16-biti?gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaJT37ruAGUgfiMOCpUW5X7Br_bSpmRLQyh1XFdPNQiv-0FqZjr_WeAaAtfeEALw_wcB) |
+| [5V 2A adapter] | Adapter | [48.72 RON](https://www.emag.ro/incarcator-pentru-nivela-laser-hilda-universal-tensiune-5v-curent-2a-5-5-x-2-5-mm-adaptor-alimentare-220v-6349/pd/DCJBSQMBM/?cmpid=101141&utm_source=google&utm_medium=cpc&utm_campaign=(RO:Whoop!)_3P-Y_%3e_Scule_electrice_and_Unelte&utm_content=82270122928&gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaL-zw6m7r1ImGxgFPLZ4aIl7b94yy3DjX_SSPFWABREixY9KfLn5SkaAqZXEALw_wcB) |
+| [Micro USB Breadboard 5V Power Supply Module] | Power Supply | [1.50 RON](https://cleste.ro/modul-alimentare-5v-micro-usb.html?utm_medium=GoogleAds&utm_campaign=&utm_source=&gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaI92As6Cx0y3WUm-b6heH19tuHWJqejoDrJV4l1LMAQLVLf7T4gvMsaAvLPEALw_wcB ) |
+| [5m 30AWG Silicone wires] | Connections | [19.50 RON](https://www.emag.ro/cablu-siliconic-ultraflexibil-14awg-bifilar-paralel-rosu-negru-cp14awg/pd/D1F6NPYBM/?cmpid=101180&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Iluminat_and_electrice&utm_content=76376892625&gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaLcDtf57lmj_iKgRz8embK0GAJY_Yc9ChU55L_7D7wNrAMA6JexHKgaAmeeEALw_wcB) |
+| [Perforated board] | Board | [75.42 RON](https://www.amazon.com/Double-Sided-Universal-Packs-12x18/dp/B08WJBX98F/ref=sr_1_2?crid=1389XD9ISH6EM&dib=eyJ2IjoiMSJ9.WsT-bgUR7rApIplnTIF8yABGZYgww0JAoqa8-54l1TNR355lFtbswumX6EjLYdIOuB_SDprsUzDVMyCV5Sf3ndWkgS4XmhOP8HSoGdqktLGlJqg9Kh14NEdeJAT_JswO9_-zVhuHUjgOttFwD51niWmp1Ki8zQtB2HBQBP7-cPFg__v6N7uQ8rY8OwfxeY_ksj0Y3gb3BFI9VtQ0sBsLcDSK-MZGQrwlJe_i-XfpMM4.4ACAhff_DsTRrHhx7lc8inTj4yOSTxPrQVeq5I2V4hs&dib_tag=se&keywords=12%2Bx%2B18%2Bcm%2BUniversal%2BPCB%2BPrototype%2BBoard%2BDouble-Side&qid=1712244151&sprefix=12%2Bx%2B18%2Bcm%2Buniversal%2Bpcb%2Bprototype%2Bboard%2Bdouble-side%2Caps%2C329&sr=8-2&th=1) |
+| [Watering Pump] | Watering the system | [26.13 RON](https://www.emag.ro/pompa-de-apa-kruger-meier-sturmkraft-220jk-x1-220l-h-3-8-x-3-7-x-2-8-cm-negru-5903900042987/pd/DTFJ05MBM/?X-Search-Id=7b217b870d8e7e7ff1d5&X-Product-Id=113031364&X-Search-Page=1&X-Search-Position=32&X-Section=search&X-MB=0&X-Search-Action=view) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```

Comment:
Do you need these two libraries?

Reply:
I need them for displaying the specific emoji on the LCD screen.